.github/workflows/tarball.yml

```diff
@@ -0,0 +1,15 @@
+name: Tarball evaluation checks
+
+permissions: read-all
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-latest
    if: github.repository_owner == 'NixOS'
```

Reply:
I'm not sure that's really needed, I'd like my forks to check for evaluation errors before I make a PR.

Reply:
Add a comment then as this will be the only one without the restriction.